common.gypi

```diff
@@ -38,7 +38,7 @@
         'V8_BASE': '<(PRODUCT_DIR)/libv8_base.a',
       }, {
         'OBJ_DIR': '<(PRODUCT_DIR)/obj.target',
-        'V8_BASE': '<(PRODUCT_DIR)/obj.target/deps/v8/tools/gyp/libv8_base.a',
+        'V8_BASE': '<(PRODUCT_DIR)/obj.target/deps/spidershim/libspidershim.a',
```

Comment:
This is not quite right, since it breaks v8 based builds.  I think this commit is probably obsolete with 518ddfc1.